documents/affiliated_package_review_guidelines.md

```diff
@@ -1,9 +1,22 @@
 # Reviewing affiliated packages
 
-This document describes the set of review criteria for packages applying to
-become Astropy affiliated.
+If you are reading this because you have accepted to review an affiliated
+package submission, thank you for taking the time to do this!
 
-Note that we use the following color-coding, which also determines if a package is accepted after its first review:
+Note that unlike for a paper where the reviews from the referee are passed on to
+the authors, one of the coordinators will also review the package and will
+create a combined report, so the report you write will not be seen by the
```

Comment:
"will not" -> "may not"?  That is, we *may* decide sometimes to just pass it on.